graphqlapi/src/main/java/io/stargate/graphql/web/Server.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.graphql.web;
+
+import io.dropwizard.Application;
+import io.dropwizard.cli.Cli;
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+import io.dropwizard.util.JarLocation;
+import io.stargate.auth.AuthenticationService;
+import io.stargate.core.metrics.api.Metrics;
+import io.stargate.db.Persistence;
+import io.stargate.graphql.GraphqlActivator;
+import io.stargate.graphql.web.resources.GraphqlDdlResource;
+import io.stargate.graphql.web.resources.GraphqlDmlResource;
+import io.stargate.graphql.web.resources.PlaygroundResource;
+import java.util.EnumSet;
+import javax.servlet.DispatcherType;
+import javax.servlet.FilterRegistration;
+import org.eclipse.jetty.servlets.CrossOriginFilter;
+import org.glassfish.hk2.utilities.binding.AbstractBinder;
+import org.osgi.framework.Bundle;
+import org.osgi.framework.FrameworkUtil;
+
+public class Server extends Application<ApplicationConfiguration> {
```

Comment:
nit: maybe name it for example `DropwizardServer` to prevent name clash between this class and `org.eclipse.jetty.server.Server`?

Reply:
They're different packages, how would they clash? This is also consistent the other modules.

Reply:
Oh, I see what you mean. The fully qualified down below.

Reply:
Good thing that you mentioned that because I had forgotten to change it.